.github/workflows/publish-release.yml

```diff
@@ -5,6 +5,8 @@ on:
     secrets:
       NPM_TOKEN:
         required: true
+      SLACK_WEBHOOK_URL:
```

Comment:
We'll need to pass this secret through explicitly each place this workflow is called as well I believe